src/main/resources/anniv5Resources/localization/eng/serpentinepack/Cardstrings.json

```diff
@@ -0,0 +1,36 @@
+{
+"${ModID}:ApplyPoison": {
+    "NAME": "Apply Poison",
+    "DESCRIPTION": "Enter ${ModID}:Venemous. NL At the start of your next turn, exit ${ModID}:Venemous."
+},"${ModID}:Guile": {
+    "NAME": "Guile",
+    "DESCRIPTION": "Draw !M! cards. NL Discard 2 cards. NL Enter ${ModID}:Cunning."
```

Comment:
If somebody modifies magic number this is not accurate. Either reword it to "Discard 1 less card than drawn" (kinda jank with no draw power) or just hardcode 2 in the card's use instead of magic-1